src/options.rs

```diff
@@ -123,6 +123,11 @@ pub struct BuildOptions {
     /// The attribute takes a default value `false`, ensuring that by default,
     /// the coverage option will be disabled).
     pub coverage: bool,
+
+    /// Dead code is linked by default to prevent a potential error with some
+    /// optimized targets. This flag allows you to opt out of it.
+    #[structopt(long = "strip-dead-code")]
```

Comment:
nitpick: `structopt` should infer this `long` name, so this line shouldn't be necessary here.

Reply:
I am not clear why, but without specifying the name, the `run_with_crash` test fails, but only when it's run as part of the full test suite.

Reply:
Ah, I figured out the issue I was running into. Because the test ids (and therefore directories) are non-deterministic, on repeated runs, I the `run_*_with_crash` tests can end up looking at the wrong output.

Running `cargo clean` between tests is a workaround, but I'll open another PR to make the project directories deterministic, as the failures are very confusing.